### DISCUSSION TITLE
How to use test.todo() in Jest | First Sutham

### DISCUSSION THREAD

**Discussion by giscus** [heyfirst/heyfirst.co#10] [General]
# How to use test.todo() in Jest | heyfirst.co

https://heyfirst.co/blog/how-to-use-test-todo-in-jest

**Comment by siriwatknp** [comment]
👍 Good tips.

**Comment by testacode** [comment]
You can also leave the `test('should do something', () => {})` without `todo` or `skip` and it will pass and you can attack it later. Why would you add anything?

**Reply by heyfirst** [reply]
So when you make the test pass with nothing, it's easy to forget about it. For example, when you write on 4-5 test files in the same time. Nothing can help you remind what's left? 

I would recommend to not write empty passed test.
